apps/aehttp/test/aehttp_integration_SUITE.erl

```diff
@@ -3845,6 +3845,55 @@ query_state_(ConnPid, <<"json-rpc">>) ->
         ConnPid, #{ <<"method">> => <<"channels.get.offchain_state">>
                   , <<"params">> => #{} })}.
 
+sc_ws_get_contract_assert_equal(ConnPid1, ConnPid2, Pubkey, Config) ->
+    ContractState = sc_ws_get_contract(ConnPid1, Pubkey, Config),
+    ContractState = sc_ws_get_contract(ConnPid2, Pubkey, Config),
+    ContractState.
+
+sc_ws_get_contract(ConnPid, Pubkey, Config) ->
+    PubkeyE = aehttp_api_encoder:encode(contract_pubkey, Pubkey),
+    {ok, #{<<"contract">> := ContractE,
+           <<"contract_state">> := ContractStateE }} = query_contract(ConnPid, PubkeyE, Config),
+    #{ <<"id">>          := IdE
+    , <<"owner_id">>     := OwnerIdE
+    , <<"vm_version">>   := VMVersion
+    , <<"abi_version">>  := CTVersion
+    , <<"log">>          := Log
```

Comment:
I am checking whether exposing this field log makes sense. See also https://github.com/aeternity/aeternity/pull/2201

Reply:
This field here will always be empty binary, I suggest at least removing this field from this API as misleading. ([More detailed description](https://www.pivotaltracker.com/story/show/164539296).) Does user of this API - or other message - expect logs? If so, I expect this PR needs to extract those from calls.

Reply:
Ok, I didn't have understanding of this field when making the PR. I think we will remove it from API for now and add it correctly if there is a need for it.